persist/sqlite/init.sql

```diff
@@ -9,6 +9,7 @@ CREATE TABLE blocks (
 	parent_id BLOB NOT NULL,
 	nonce BLOB NOT NULL,
 	timestamp INTEGER NOT NULL,
+	chain_index_element BLOB NOT NULL,
```

Comment:
I don't think you need to store the full element in here since you already got the block id and height. Maybe only the leaf index and proof are missing?

Maybe @lukechampine can verify this assumption.